### DISCUSSION TITLE
Thanks !

### DISCUSSION THREAD

**Discussion by sander76** [SukramJ/aiohomematic#536] [General]
@danielperna84 I have been looking how it is best to contact you directly.

Just want to say : Thanks !

The new integration works like a charm.

Sander.

**Comment by danielperna84** [comment]
You have to thank @SukramJ . He's the one doing the incredible developing here now. 👍

**Reply by sander76** [reply]
Allright. Thanks @SukramJ ! Really appreciate all the work you're putting into it. I am slowly moving over all my homematic-ip cloud stuff to this integration !
